src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2085 @@
+  |**SENDMESSAGE**|**X**|
+  |**SCHEDULEMESSAGE**|**X**|
+
+Program rules are evaluated in the importer in the same way they are evaluated in the Tracker apps. To summarize, the following conditions are considered when enforcing the program rules:
+* The program rule must be linked to the data being imported. For example a program stage or a data element.
+* The Program rule's condition must be evaluated to true
+
+The results of the program rules depends on the actions defined in those rules:
+* Program rule actions may end in 2 different results: Warnings or Errors.
+  * Errors will make the validation fail, while the warnings will be reported as a message in the import summary.
+    * SHOWWARNING and WARNINGONCOMPLETION actions can generate only Warnings.
+    * SHOWERROR, ERRORONCOMPLETION and SETMANDATORYFIELD actions can generate only Errors.
+    * ASSIGN action can generate both Warnings and Errors.
+      * When the action is assigning a value to an empty attribute/data element, a warning is generated.
+      * When the action is assigning a value to a an attribute/data element that has already the same value to be assigned, a warning is generated.
+      * When the action is assigning a value to a an attribute/data element that has already a value and the value to be assigned is different, an error is generated, unless `RULE_ENGINE_ASSIGN_OVERWRITE` system setting is set to true.
+
+Additionally, program rules can also result in side-effects, like send and schedule message. More information about side-effects can be found in the following section.
+
+### Side Effects
+
+<!--DHIS2-SECTION-ID:webapi_nti_side_effects-->
+
+After an import has been completed, specific tasks might be triggered as a result of the import. These tasks are what we refer to as "Side effects". These tasks perform operations that do not affect the import itself.
+
+Side effects are tasks running detached from the import but are always triggered by an import. Since side effects are detached from the import, they can fail even when the import is successful. Additionally, side effects are only run when the import is successful, so they cannot fail the other way around.
+
+The following side effects are currently supported:
+  
+  |Side Effects|Supported|Description|
+  |---|:---:|---|
+  |**Tracker Notification**|**X**| Updates can trigger notifications. Updates which trigger notifications are **enrollment**, **event update**, **event or enrollment completion**. |
+  |**ProgramRule Notification**|**X**| Program rules can trigger notifications. Note that these notifications are part of program rule effects which are generated through the DHIS2 rule engine.|
+  
+  > ***NOTE*** 
+  > Certain configurations can control the execution of side effects. `skipSideEffects` flag can be set during the import to skip side effects entirely. This parameter can be useful if you import something you don't want to trigger notifications for, as an example.
```

Comment:
There's also skipRuleEngine described in jira [here](https://jira.dhis2.org/browse/DHIS2-9269) that I don't see mentioned in program rules section

Reply:
I added it in the list of params I added after you reviewed. Will add a note in the program rule section as well